FluentTc/Locators/MoreOptionsHavingBuilder.cs

```diff
@@ -8,13 +8,15 @@ public interface IMoreOptionsHavingBuilder
         IMoreOptionsHavingBuilder AsPersonal();
         IMoreOptionsHavingBuilder WithComment(string comment);
         IMoreOptionsHavingBuilder OnBranch(string branchName);
+        IMoreOptionsHavingBuilder OnChange(long changeId);
```

Comment:
@borismod, thank you for your feedback. I added a new interface IOnChangeHavingBuilder for scalability. The use really became more convenient.

```C#
    var build = client.RunBuildConfiguration(
        hc => hc.Id(configuration.Id),
        a => a.Id(agent.Id),
        paremeters =>
            {
                customProperties.ForEach(p => paremeters.Parameter(p.Name, p.Value));
            },
        options =>
            options .OnBranch("release").WithCleanSources().OnChange(change => change.Id(changeId))
```